cli/config/configfile/file.go

```diff
@@ -190,15 +191,26 @@ func (configFile *ConfigFile) Save() error {
 	if err != nil {
 		return err
 	}
+	defer func() {
+		temp.Close()
+		if retErr != nil {
+			if err := os.Remove(temp.Name()); err != nil {
```

Comment:
Not sure if there's a situation where that's possible (thinking if an error occurs on `os.Rename()`; not sure how atomic that is); should we ignore errors if the file didn't exist?

```suggestion
			if err := os.Remove(temp.Name()); err != nil && !os.IsNotExist(err) {
```

Guess it's not needed 😅